src/call.c

```diff
@@ -2212,17 +2233,16 @@ int call_accept(struct call *call, struct sipsess_sock *sess_sock,
 		mem_deref(rid);
 	}
 
-	err = sipsess_accept(&call->sess, sess_sock, msg, 180, "Ringing",
+	err = sipsess_accept(&call->sess, sess_sock, msg, 180, "Trying",
```

Comment:
180 Trying? That seems unusual. What's the reasoning behind this?

Reply:
I guess @juha-h wants to reply with 100 Trying and put the 180 Ringing into the app/module menu. But what I remember is that 100 Trying is sent already maybe internally from `re`. We have to be sure not to send it twice.